### PR TITLE
readme documentation for application specific properties

### PR DESCRIPTION
**Pull request by weiweishi**

### CODE REVIEW

File: data_dictionary/readme_application_properties.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  - predicate: N/A (Solr Only)
+  - data: derived from data in attribute: prism: doi in Fedora
+  - multivalue: no
+  - reason of use: to enable search and present doi link in the proper URL structure - https://doi.org/10.7939/R39J1J
+- item_type_with_status
+  - format: Articles_publication_status
+  - used for: searching, faceting and forms for this combined attribute
+  - predicate: N/A (Solr Only)
+  - data: derived from dcterms:type + bibo:status
+  - multivalue: yes
```
[the remark this answers is not in the document]
Comment (cwant) [reply data_dictionary/readme_application_properties.md]: Everything looks good except `multivalue` (should be single valued as far as I know). This is not only for articles though, so the `format` might be misleading (for other item types, this holds only the item type, not the publication status).

File: data_dictionary/readme_application_properties.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  - predicate: http://terms.library.ualberta.ca/identifiers/path
+  - multivalue: no
+  - reason of use: to avoid multiple calls to solr when fetching the community information, and to keep it consistent with member_of_paths so the facet results can be coalesced with Items
+
+- community_title
+  - format: string 
+  - used for: display community title on collection related UI
+  - predicate: N/A (Solr Only)
+  - data: derived from UAL:community_id
+  - reason of use: to avoid additional calls to solr to retrieve the title whenever the information is need.
```
[the remark this answers is not in the document]
Comment (mbarnett) [reply data_dictionary/readme_application_properties.md]: Ah yeah, I misread that. community_title is cached in solr to reduce the overhead of rendering collections.

**Comment by zschoenb** [comment]
Everyone happy with the PR? I'm going to close at the end of the day. @weiweishi @mbarnett @murny @cwant @sfarnel @danydvd

**Comment by cwant** [comment]
Hi Zach,

I guess for the sake of completeness, these should also be included:

https://github.com/ualbertalib/jupiter/pull/330

**Comment by weiweishi** [comment]
I have an update sitting on my workstation. I will try to push it before
the end of the day.

*Weiwei Shi*

Digital Initiative Applications Librarian
University of Alberta Libraries
2-10L Cameron Library
Edmonton, Alberta, Canada T6G 2J8
Phone:(780)492-7802
Fax: (780)248-1209
Email: weiwei.shi@ualberta.ca

File: data_dictionary/readme_application_properties.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  - multivalue: yes
+  - reason of use: to avoid multiple calls to Solr when fetching the collection/community information for item view or search facets. 
+
+- doi_without_label
+  - format: 10.7939/R39J1J
+  - used for: present doi link on the user interface
+  - predicate: N/A (Solr Only)
+  - data: derived from data in attribute: prism: doi in Fedora
+  - multivalue: no
+  - reason of use: to enable search and present doi link in the proper URL structure - https://doi.org/10.7939/R39J1J
```
[the remark this answers is not in the document]
Comment (weiweishi) [reply data_dictionary/readme_application_properties.md]: This doc is to document defined in the application but not defined in the data dictionary, so I don't think there's need to justify indices for properties that are in the data dictionary. Normally reasons for having these solr-only indices will be documented in the code for the dev team, but they may not be obvious for metadata team or other people who don't work with the code on the regular basis. This hopefully will provide them a complete picture of what is available in the application.

**Comment by weiweishi** [comment]
@zschoenb updated the documentation and incorporated comments into the PR. Ready for your review and approval.

File: data_dictionary/readme_application_properties.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+## Item
+
+### Properties available in application only
+
+- member_of_paths
+  - format: community_id/collection_id
+  - used for: breadcrumbs, community/collection facets, item view, deposit form
+  - predicate: http://terms.library.ualberta.ca/identifiers/path
+  - multivalue: yes
+  - reason of use: to avoid multiple calls to Solr when fetching the collection/community information for item view or search facets.
```
[the remark this answers is not in the document]
Comment (zschoenb) [reply data_dictionary/readme_application_properties.md]: predicate: http://terms.library.ualberta.ca/identifiers/path
"identifiers" isn't in our namespace anymore.
It should be `http://terms.library.ualberta.ca/path`

**Pull request merged**